sources/application/UI/Base/Layout/NavigationMenu/NavigationMenu.php

```diff
@@ -290,6 +289,10 @@ public function ComputeAppIconLink(): void
 		}
 	}
 
+	private function IsOrgMenuFilterAllowed() : bool {
```

Comment:
```suggestion
	/**
	 * @return True if the silo selection is enabled, false otherwise
	 * @since 3.1.0
	 */
	public function IsSiloSelectionEnabled() : bool {
```
Rename: Consistency with other methods/properies such as `$bHasSiloSelected`
Visibility: That way the method can be used in TWIG and such. Even though we tend to use defensive programming in some parts of iTop, we rather be dev-friendly in others.